### DISCUSSION TITLE
Songpress 1.8.1 RC1

### DISCUSSION THREAD

**Discussion by lallulli** [lallulli/songpress#57] [General]
This bugfix candidate release fixes some issues:

- A problem with find-replace (#53) 
- An error on application exit (#54)
- An internationalization problem with some languages (#55)

<hr /><em>This discussion was created from the release <a href='https://github.com/lallulli/songpress/releases/tag/'>Songpress 1.8.1 RC1</a>.</em>

**Comment by Maxrella** [comment]
I tried to set the font of the lyrics and chords in a format different from the standard one but the transcription of the song remains in the standard format. How can I make the settings work correctly? Thank you
